src/InstrumentationEngine.WindowsAzureDiagnostics.NuGet/InstrumentationEngine.WindowsAzureDiagnostics.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<Project>
+  <PropertyGroup>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <TargetExt>.nupkg</TargetExt>
+    <PackageId>Microsoft.InstrumentationEngine.WindowsAzureDiagnostics</PackageId>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'EnlistmentRoot.marker'))\build\Common.props" />
+  <Import Project="$(EnlistmentRoot)\build\Packaging.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <PropertyGroup>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <ResolveNuGetPackages>true</ResolveNuGetPackages>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <NuspecFile>InstrumentationEngine.WindowsAzureDiagnostics.nuspec</NuspecFile>
```

Comment:
>WindowsAzureDiagnostics [](start = 38, length = 23)

While it's unlikely that anyone other than WAD will consume this Nuget package, the package could be more generically named.

Reply:
I guess I can remove `Microsoft` from  `Microsoft.InstrumentationEngine.WindowsAzureDiagnostics`

---
In reply to: [337094437](https://github.com/microsoft/CLRInstrumentationEngine/pull/150#discussion_r337094437) [](ancestors = 337094437)

Reply:
Discussed offline, although this package is consumed by WAD, its contents is just the installer and should be more generically named. 

If we ever have more specific bits or customization that is WAD-specific, then we can name this with WindowsAzureDiagnostics.

---
In reply to: [337098292](https://github.com/microsoft/CLRInstrumentationEngine/pull/150#discussion_r337098292) [](ancestors = 337098292,337094437)